en/FAQ.md

```diff
@@ -0,0 +1,39 @@
+# DiversiFAQ: Questions and answers about diversity in technology
+
+### What benefits has a diverse team?
+
+The software we develop inherits our biases. There are many [documentede cases](https://www.theguardian.com/technology/2017/apr/13/ai-programs-exhibit-racist-and-sexist-biases-research-reveals) about this.
```

Comment:
I think we have a typo in `documentede cases`, right? 😄

Reply:
Fixed 🙏 https://github.com/folkswhocode/diversifaq/pull/9/commits/f6784f1f15e1518b6eaeb87dc6ae6edba4171f1a